### DISCUSSION TITLE
Texture quality setting

### DISCUSSION THREAD

**Discussion by GazziFX** [SmartlyDressedGames/Unturned-3.x-Community#3723] [Ideas]
I have some players on my server with old GPUs (NVIDIA GT250). And I think it's better to have quality settings instead of reducing resolution of all my textures.

They have a black screen when connecting to the server and the following message in the logs, I assume they ran out of VRAM.
```
D3D11: Failed to create RenderTexture (1280 x 1024 fmt 9 aa 1), error 0x8007000e
```
I don't really understand what the textures are because LevelBatching is turned off. Is there any solutions?

**Comment by SDGNelson** [comment]
Error 0x8007000e is E_OUTOFMEMORY (https://learn.microsoft.com/en-us/windows/win32/direct3d11/d3d11-graphics-reference-returnvalues)

1280 x 1024 looks like a display resolution rather than a regular texture resolution, so it is probably generic out of VRAM as you mentioned.
